src/ui/src/builder/BuilderVault.spec.ts

```
@@ -0,0 +1,72 @@
+		});
+	}
+
+	let windowOpen: Mock;
+	beforeEach(() => {
+		mockCore.writerApplication.value = {
+			id: "app-lication-uuid",
+			organizationId: "1",
+		};
+		windowOpen = vi.fn();
+		window.open = windowOpen;
+	});
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)